README.md

```diff
@@ -44,6 +44,21 @@
     pip3 install --upgrade cid-cmd
     ```
 
+5. Deploy the Dashboards
+    ```bash
+    cid-cmd deploy
+    ```
+
```

Comment:
Maybe let's add a section here Optional Steps with update and help? having them in a sequence with deploy might mislead a bit